src/test/setupTests.js

```diff
@@ -0,0 +1,8 @@
+// import { configure } from 'enzyme';
+// import Adapter from 'enzyme-adapter-react-16';
+
+//TODO: switch to import
+let Adapter = require('enzyme-adapter-react-16');
+let Enzyme = require('enzyme');
```

Comment:
Better to use `const` for immutability:, and you can require `configure` immediately as no need to import all `Enzyme` here (module tree shaking):
```suggestion
const {configure} = require('enzyme');
```